mpas_analysis/ocean/ocean_modelvsobs.py

```diff
@@ -41,7 +59,7 @@ def ocn_modelvsobs(config, field):
 
     #Seems like the following line should be a config.get option and not
     #read every time series file when only a subset is taken
-    infiles = "".join([indir,"/am.mpas-o.timeSeriesStats.????-*.nc"])
+    #infiles = "".join([indir,"/am.mpas-o.timeSeriesStats.????-*.nc"])
```

Comment:
@pwolfram, I agree.  I was keeping them to help me debug for the time being.  Please don't worry about reviewing the code right now.  As I just posted, it's very much a work in progress based on @milenaveneziani's and my recent discussion of what features are needed.